src/persist-client/examples/inspect.rs

```diff
@@ -82,15 +82,15 @@ pub struct StateArgs {
     ///     &options=--search_path=consensus
     ///
     #[clap(long, verbatim_doc_comment)]
-    consensus_uri: String,
+    pub(crate) consensus_uri: String,
```

Comment:
while you're here, would you mind creating equivalent env vars to set the consensus and blob uris? we'll want env vars so we're not passing around secrets in the Kube pod spec